task_2/variable_var_let.ts

```diff
@@ -38,6 +38,8 @@ let secondVariable: boolean;
 firstVariable = "Hello World" // initialise variable
 secondVariable = true;
 
+//once you assign a value to a constant variable, you cannot change it later
+
```

Comment:
@VitaliiMalatok 
let- can be defined or initialised; the value can be reassigned
const - always is initialised; the value can't be changes

Reply:
@VitaliiMalatok did you fix it?

Reply:
fixed